### PR TITLE
fix issue causing ripple shows strangely in Chrome version 51

### PR DESCRIPTION
**Pull request by cyx8808**
to fix issue [#8815](https://github.com/angular/material/issues/8815)

### CODE REVIEW

File: src/core/services/ripple/ripple.js [hunk 1]
```diff
@@ -432,6 +432,7 @@ InkRippleCtrl.prototype.removeRipple = function (ripple) {
   if (index < 0) return;
   this.ripples.splice(this.ripples.indexOf(ripple), 1);
   ripple.removeClass('md-ripple-active');
+  ripple.addClass('md-ripple-remove');
```
Comment (crisbeto) [inline src/core/services/ripple/ripple.js]: Shouldn't this class be removed at some point? Otherwise it will always stick around after the first click.

**Comment by cyx8808** [comment]
I signed it!

Date: Wed, 22 Jun 2016 06:04:49 -0700

File: src/core/services/ripple/ripple.js (hunk 1 above)
Comment (cyx8808) [inline src/core/services/ripple/ripple.js]: the ripple itselt will be removed soon after

**Comment by ThomasBurleson** [comment]
@crisbeto - can you validate this?

**Comment by crisbeto** [comment]
Tested it out and LGTM @ThomasBurleson, it seems to help with a similar issue with Edge as well.

**Pull request closed without merging**